src/xcode/ENA/ENA/Source/Extensions/UIColor+ColorStyle.swift

```diff
@@ -45,7 +45,7 @@ public enum ColorStyle: String {
 	case shadow
 }
 
-extension UIColor {
+public extension UIColor {
```

Comment:
```suggestion
extension UIColor {
```

Reply:
This was changed because without this every screen that uses ```ENAButton``` is crashing in IB. It's in the description.